gallery/src/components/NxSmallThreatCounter/NxSmallThreatCounterInfiniteMaxDigitsExample.tsx

```diff
@@ -15,15 +15,15 @@ export default function NxSmallThreatCounterInfiniteMaxDigitsExample() {
                               criticalCount={1234567890}
                               severeCount={5}
                               moderateCount={1337}
-                              lowCount={2352}
-                              noneCount={0}
+                              lowCount={0}
+                              noneCount={2353}
                               unspecifiedCount={1} />
       </div>
       <div style={{ marginTop: '16px' }}>
         <NxSmallThreatCounter maxDigits={Infinity}
                               criticalCount={2}
                               severeCount={50}
-                              noneCount={0}
+                              noneCount={23}
```

Comment:
Out of curiosity, it is necessary to change the number here?

Reply:
> Out of curiosity, it is necessary to change the number here?

Yeah the `zero` state was overriding the text color of the none count, so I changed the number to avoid any problems

Reply:
> the zero state was overriding the text color of the none count

If I remember correctly in this use case, shouldn't the threat color remain the same even if there is a value of 0?

Reply:
> > the zero state was overriding the text color of the none count
> 
> If I remember correctly in this use case, shouldn't the threat color remain the same even if there is a value of 0?

I totally missed that initially, thanks for catching it!  Updated [db368ae](https://github.com/sonatype/sonatype-react-shared-components/pull/1071/commits/db368ae238807d20b1332a67ada66dccc0e0135b), [062ca2d](https://github.com/sonatype/sonatype-react-shared-components/pull/1071/commits/062ca2d226c41a55fa754226e9282e32348749f6)